lib/tasks/images.js

```diff
@@ -85,11 +85,18 @@ module.exports = function(gulp, config) {
 			var transformStream = new stream.Transform({objectMode: true});
 			transformStream._transform = function (file, encoding, callback) {
 				if(file.imageTransform) {
-					sharp(file.contents).resize(file.imageTransform.width, file.imageTransform.height).withoutEnlargement().jpeg(file.imageTransform.options).toBuffer(function (err, buffer, info) {
-						if(err) console.error(err)
-						file.contents = buffer
-						callback(null, file)
-					})
+					jimp
+						.read(file.contents)
+						.then(function(image) {
+							image
+								.resize(file.imageTransform.width, file.imageTransform.height)
+								.quality(image.quality)
```

Comment:
From what I understand it will fail if undefined ([source](https://github.com/oliver-moran/jimp/blob/master/index.js#L683-L684)).

Reply:
default is set at 90 few lines above

Reply:
(but was undefined anyway, thx for the sharp eye!)